internal/coreconfig/coreconfig.go

```diff
@@ -209,6 +209,8 @@ var (
 	PluginsIdentityList = ffc("plugins.identity")
 	// DebugPort a HTTP port on which to enable the go debugger
 	DebugPort = ffc("debug.port")
+	// DebugAddress the HTTP interface for the debugger to listen on
+	DebugAddress = ffc("debug.address")
```

Comment:
Think you'll need to add a documentation entry for this one too.

Reply:
Good point, let me do that

Reply:
Here's where to add the config doc:
https://github.com/hyperledger/firefly/blob/892ebae4d2202a4abc59aa056aaffbe8849cd456/internal/coremsgs/en_config_descriptions.go#L219

Then `make reference` before you check in.